packages/xo-web/src/xo-app/xostor/new-xostor-form.js

```diff
@@ -422,10 +422,10 @@ const SummaryCard = decorate([
                 <Col size={6}>{_('keyValue', { key: _('provisioning'), value: state.provisioning.label })}</Col>
               </Row>
               <Row>
-                <Col size={6}>{_('keyValue', { key: _('pool'), value: <PoolRenderItem id={state.poolId} /> })}</Col>
-                <Col size={6}>
+                <Col size={12}>{_('keyValue', { key: _('pool'), value: <PoolRenderItem id={state.poolId} /> })}</Col>
+                {/* <Col size={6}>
```

Comment:
Why?

Reply:
@Rajaa-BARHTAOUI Because it's not implemented in XCP-ng yet (cf PR description)
@MathieuRA Will it be implemented before `xostor/master` is merged into `master`?

Reply:
Probably not.